apps/artsy_primer/templates/article.jade

```diff
@@ -0,0 +1,20 @@
+extends ../../../components/main_layout/templates/blank
+
+append locals
+  - assetPackage = 'misc'
+
+block body
+  header.artsy-primer-header
+    .responsive-layout-container
+        .artsy-primer-logo
+          include ./artsy_primer_logo.svg
+        h2.artsy-primer-h2 Portfolio
+        h3.artsy-primer-h3 Your personalized collection of available artwork.
+        .artsy-primer-user
+          include ../../../components/main_layout/header/templates/user
+
+  .artsy-primer-article
+    include ../../../components/article/templates/index
+
+  .responsive-layout-container: .mlf.mlf__fullwidth
+    include ../../../components/main_layout/footer/lower
```

Comment:
Below you can see that I've broken up the main layout templates so I can more easily re-use the individual pieces to construct a stripped down form of it in the screenshot embedded in the description.